### PR TITLE
Quiz EN new content level 8 and 9

### PR DESCRIPTION
**Pull request by MarleenGilsing**
Quiz EN new content level 8 and 9

### CODE REVIEW

**Review by Felienne — APPROVED** [review]

File: content/quizzes/en.yaml [hunk 1]
```diff
@@ -1228,6 +1228,470 @@ levels:
...
             question_score: '10'
+    8:
+        1:
+            question_text: "Which output will be produced by this code?"
+            code: |-
+                {repeat} 2 {times}
+                    {print} 'Hello'
+                    {print} 'Im Hedy!'
+            mp_choice_options:
+                - "code": "Hello \n Im Hedy!"
```
Comment (Mark-Giesen) [inline content/quizzes/en.yaml:1239]: In de NL file heet deze tag option_text

File: content/quizzes/en.yaml [hunk 2]
```diff
@@ -1228,6 +1228,470 @@ levels:
             correct_answer: "B"
             hint: "Mind the order of the sentences."
             question_score: '10'
+    8:
+        1:
+            question_text: "Which output will be produced by this code?"
+            code: |-
+                {repeat} 2 {times}
+                    {print} 'Hello'
```
Comment (Mark-Giesen) [inline content/quizzes/en.yaml:1236]: In de NL file staan hier geen accolades om de keywords heen, niet consequent.

**Review by Mark-Giesen — CHANGES_REQUESTED** [review]
EN en NL files hebben andere manier van keywords weergeven en andere tagnamen voor dezelfde teksten.
Het is verstandig om EN alleen via deze repo toe te voegen en daarna andere talen in Weblate

**Comment by Felienne** [comment]
> EN en NL files hebben andere manier van keywords weergeven en andere tagnamen voor dezelfde teksten. Het is verstandig om EN alleen via deze repo toe te voegen en daarna andere talen in Weblate

Hi Mark! I am not sure what the concrete suggestion is. Merge this one (this is En) and then fix Nl?

And maybe we should make issues for the Nl problems? If we only mention them here I think they might get lost when we merge this.

File: content/quizzes/en.yaml [hunk 3]
```diff
@@ -529,7 +529,7 @@ levels:
                 -   "option_text": "{at} {random} now needs quotation marks"
                     "feedback": "No, but 2 other commands do."
             correct_answer: "A"
-            hint: "In level 4 you need quotation marks for 2 commands."
+            hint: "In level 4 you need quotation marks {for} 2 commands."
```
Comment (Felienne) [inline content/quizzes/en.yaml:532]:
```suggestion
            hint: "In level 4 you need quotation marks for 2 commands."
```

This for is not a command :)

File: content/quizzes/en.yaml [hunk 4]
```diff
@@ -572,7 +572,7 @@ levels:
                 -   "option_text": "You do not need quotation marks when using the {ask} command"
                     "feedback": "Both {print} and {ask} require quotation marks"
                 -   "option_text": "You can choose yourself whether to use quotation marks or not."
-                    "feedback": "Unfortunately, Hedy is stricter than that."
+                    "feedback": "Un{for}tunately, Hedy is stricter than that."
```
Comment (Felienne) [inline content/quizzes/en.yaml:575]: This too is a misplaced for, maybe do one more manual check of the replacements?

File: content/quizzes/en.yaml (hunk 1 above)
Comment (Felienne) [reply content/quizzes/en.yaml:1239]: Ah I understand now, thanks for the clarification!! I will fix this in the nl version (and fix the rest fo the PR a bit too with the misplaced {})

**Review by Felienne — CHANGES_REQUESTED** [review]

**Review by Mark-Giesen — APPROVED** [review]
I believe all is fixed to be able to merge. Some translations need to be changed via Weblate (or otherwise)

**Review by Mark-Giesen — APPROVED** [review]
OK

**Comment by Felienne** [comment]
@TiBiBa somehow, this makes an e2e test fail? I have no clue why that happens with a content change, could  you take a look?

**Comment by TiBiBa** [comment]
Should be fixed! In the e2e tests we try to visit all pages in English and there was a placeholder mistakes in the English quiz YAML file. Would be nice however to visit all pages in all languages to make sure it always works (will make an issue!).

**Pull request merged**